### PR TITLE
disable danger on builds without token

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -196,7 +196,7 @@ travis-lint-dist: travis-noop
 
 # Test steps
 travis-test-danger:
-	bundle exec danger
+	[[ -z "${DANGER_GITHUB_API_TOKEN}" ]] && echo "Skipping danger on PR." || bundle exec danger
```
Comment (mattrobenolt) [inline Makefile]: This needs to be just single brackets.

```make
travis-test-danger:
	[ -z "${DANGER_GITHUB_API_TOKEN}" ] && echo "Skipping danger on PR." || bundle exec danger
```

and in theory this is fine. The syntax you used doesn't work on Linux it seems.

**Review by mattrobenolt — CHANGES_REQUESTED** [review]

**Comment by mattrobenolt** [comment]
Alternatively, I'm not entirely sure how bad of an idea it'd be to just allow Danger to run on pull requests from outside contributors. Rather, unclear off hand what someone could do with the API key we are using.

**Comment by graingert** [comment]
Looks like they recommend making it available: http://danger.systems/guides/getting_started.html

**Comment by mattrobenolt** [comment]
I have fixed this by using a public key now. <3

**Pull request closed without merging**